service/version_bundles.go

```diff
@@ -42,13 +52,21 @@ func newVersionBundles() []versionbundle.Bundle {
 				},
 			},
 			Components: []versionbundle.Component{
+				{
+					Name:    "containerlinux",
```

Comment:
Not sure if that should be the name.

Reply:
Towards the naming I would ask @marians and @puja108 since this is customer facing. 

I think it is good to have the changelogs but I am not sure we should add the components now. You actually only want to update the version. The changelog is there. This is good. When we add CoreOS as component here we should also do this in the other operators. I think we should maybe just not do it now and keep it to the changelogs. When we later have the cloud config operator or whatever we can add the component there.

Reply:
name is fine

Reply:
I also saw that there is a docker component which the other operators do not have AFAIK. We should align this then.

Reply:
They do: https://github.com/giantswarm/kvm-operator/blob/8f35c674414d9098aa415d2d5774c01da08b44c3/service/version_bundles.go#L49-L52

This is why I started the discussion about docker version in version bundles.
@calvix says this is important to show docker version to the customer.

Reply:
Cool.

Reply:
I think I meant its not important for the customer :)

Reply:
lol